src/main/java/io/cryostat/diagnostic/Diagnostics.java

```diff
@@ -87,13 +87,13 @@ public class Diagnostics {
     @Inject EventBus bus;
     @Inject DiagnosticsHelper helper;
 
-    @Path("/threaddump")
+    @Path("targets/{targetId}/threaddump")
     @RolesAllowed("write")
     @Blocking
     @POST
     public String threadDump(
             HttpServerResponse response, @RestPath long targetId, @RestQuery String format) {
-        log.trace("Creating new thread dump request");
+        log.trace("Creating new thread dump request for target: " + targetId);
```

Comment:
Best to use the `log.tracev("message {0} {1}", obj1, obj2)` format specifier style rather than direct string concatenation. Using concatenation will result in an eager `.toString()` conversion being called on the operands, even if the message is logged at a lower level than currently configured.